salt/utils/saltminionservice.py

```diff
@@ -15,7 +15,7 @@
 import winerror
```

Comment:
I'm not following the logic here. Why are we gating Windows-specific classes in functions but then leaving windows-specific imports at the global level? In my testing, Windows-specific exceptions are still thrown. I suspect we need to put these imports into a `__virtual__` function.